# first 4 bit: Total number of actions
# last 4 bit: Total running times
uint8 action_repeat
uint8[16] command_list
int16[16] command_parameter
